source/turtles.rst

```diff
@@ -17,38 +17,39 @@ We also look at the two ways the `python3` interpreter can execute your python c
 Interactive interpreter 
 =======================
 
-We launch the python interpreter through `cmd.exe`:
+We launch the python interpreter through `terminal`:
 
-1. Press the two keys 'Windows + R' together
-2. Enter `cmd.exe` in the search prompt and enter.
+1. Press `Cmd(apple) + Spacebar` (the two keys together). A search box pops up.
+2. Type `terminal` and press enter. 
 
 A window will appear with a prompt:: 
 
-    C:\Users\greg-lo>
+    ~
 
-Type `python3` to enter the interactive shell::
+Type `python` to enter the interactive shell::
 
-    C:\Users\greg-lo>python
-    Python 3.4.2 (v3.4.2:8711a0951384, Sep 21 2014, 21:16:45) [MSC v.1600 32 b
-    it (Intel)] on win32
-    Type "help", "copyright", "credits" or "license" for more information.
+    ~ >python
+    ...
```

Comment:
I wonder whether it might be more reassuring to see that all the mumbo-jumbo numbers are meant to be there, even though in the instructions they appear confusing?